pkg/config/config.go

```diff
@@ -327,12 +338,15 @@ func load(fs *flag.FlagSet, args []string, loader func(string, bool, *Config) er
 	var (
 		cfg = DefaultConfig
 
-		printVersion    bool
-		file            string
-		configExpandEnv bool
+		printVersion             bool
+		file                     string
+		dynamicConfigurationPath string
+		configExpandEnv          bool
```

Comment:
nit: use `config` or `configuration` for both of these for consistency, but don't mix :)